clang/bindings/python/clang/cindex.py

```diff
@@ -3466,6 +3479,8 @@ def get_tokens(self, locations=None, extent=None):
         """
         if locations is not None:
             extent = SourceRange(start=locations[0], end=locations[1])
+        if extent is None:
+            raise TypeError("get_tokens() requires at least one argument")
```

Comment:
This previously returned the following error:
`ctypes.ArgumentError: argument 2: TypeError: expected SourceRange instance instead of NoneType`
from the library function since it gets passed `None`. This moves the error into `get_tokens` on Python side, gives a more explicit explanation and lets us guarantee that `TokenGroup.get_tokens()` is always passed a `SourceRange`

Reply:
Can you move this check up, and test both arguments for None? I don't like how it depends on the previous `if` to work.